### PR TITLE
Upgrade to latest ADF beta & UI fixes

### PR DESCRIPTION
**Pull request by suzanadirla**
## PR Checklist
Please check if your PR fulfills the following requirements:

```
- [x] The commit message follows our guidelines: https://github.com/Alfresco/alfresco-content-app/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
```

## PR Type
What kind of change does this PR introduce?

```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[x] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Application / Infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?
Besides ADF upgrade, added also with this PR are:
- the UI fixes for issues [ACA-2363](https://issues.alfresco.com/jira/browse/ACA-2363) and [ACA-2331](https://issues.alfresco.com/jira/browse/ACA-2331)
 - the revert of  the code changes done because of issue [ACA-2350](https://issues.alfresco.com/jira/browse/ACA-2350)

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/app/ui/overrides/adf-document-list.theme.scss [hunk 1]
```diff
@@ -123,5 +99,25 @@
     padding-top: 9px;
     padding-bottom: 9px;
     min-height: auto;
+
+    .adf-datatable-cell-value {
```
Comment (DenysVuika) [inline src/app/ui/overrides/adf-document-list.theme.scss]: can this be moved to ADF?

File: src/app/ui/overrides/adf-document-list.theme.scss [hunk 2]
```diff
@@ -123,5 +99,25 @@
...
+      display: inline-block;
+      padding-left: 0;
+    }
+  }
+
+  .adf-content-node-selector-content-list .adf-datatable-list {
+    .adf-datatable-cell {
```
Comment (DenysVuika) [inline src/app/ui/overrides/adf-document-list.theme.scss]: this is not app-specific thing and should be fixed at ADF side.

File: src/app/ui/overrides/adf-document-list.theme.scss [hunk 3]
```diff
@@ -76,10 +50,12 @@
 
     .adf-datatable-cell-header,
     .adf-datatable-cell {
-      padding: 0 2px;
-      box-sizing: border-box;
       min-width: 120px;
 
+      &.adf-no-grow-cell {
+        min-width: 120px;
```
Comment (DenysVuika) [inline src/app/ui/overrides/adf-document-list.theme.scss]: not app-specific style, should be in ADF

Comment (suzanadirla) [reply src/app/ui/overrides/adf-document-list.theme.scss]: true, all these styles will be moved on ADF side with issue [ACA-2361](https://issues.alfresco.com/jira/browse/ACA-2361)

File: src/app/ui/overrides/adf-document-list.theme.scss (hunk 1 above)
Comment (suzanadirla) [reply src/app/ui/overrides/adf-document-list.theme.scss]: yes, the plan is to move all these styles to ADF with [ACA-2361](https://issues.alfresco.com/jira/browse/ACA-2361)

**Pull request merged**